infra/base-images/base-runner/run_fuzzer

```diff
@@ -96,18 +96,27 @@ if [ -f $OPTIONS_FILE ]; then
 fi
 
 if [[ "$FUZZING_ENGINE" = afl ]]; then
-  # https://chromium.googlesource.com/chromium/src/+/master/third_party/afl/src/docs/env_variables.txt
+
+  # Set afl++ environment options.
   export ASAN_OPTIONS="$ASAN_OPTIONS:abort_on_error=1:symbolize=0"
   export MSAN_OPTIONS="$MSAN_OPTIONS:exit_code=86:symbolize=0"
   export UBSAN_OPTIONS="$UBSAN_OPTIONS:symbolize=0"
   export AFL_I_DONT_CARE_ABOUT_MISSING_CRASHES=1
   export AFL_SKIP_CPUFREQ=1
   export AFL_NO_AFFINITY=1
+  export AFL_FAST_CAL=1
   export AFL_MAP_SIZE=4194304
+
+  #
```

Comment:
This can be removed as these runtime options will be added on ClusterFuzz side.
This place is just a litmus test for folks to execute fuzzer locally after writing it, this part is best to leave it predictable.